SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorConfiguration.java

```diff
@@ -188,6 +188,11 @@
   @JsonProperty
   private boolean failTaskOnInvalidArtifactSignature = true;
 
+  @JsonProperty
+  private boolean failOnArtifactWithNoMatchingSignature = false;
+
+  private boolean failOnSignatureWithNoMatchingArtifact = false;
```

Comment:
Why doesn't this field need the `@JsonProperty` annotation?

Reply:
Not sure, just moved that one up here to be with the others.